### ISSUE TITLE
No Interface

### ISSUE THREAD

**Issue by ToxicTerror86** [decodinatorX/Liroth-Revamped-Forge#24]
Neither the Liroth Splitter or Quantum Extractor has an interface to input the neccessary items to use them, also they dont drop when broken even with silk touch

**Comment by decodinatorX** [comment]
What is the MC version and Mod version you are playing with?

**Comment by ToxicTerror86** [comment]
Minecraft 1.20.1 and the newest release of Liroth Revamped

**Comment by ToxicTerror86** [comment]
To follow up it's Liroth_R_1.3.5.8_1.20.jar

**Comment by decodinatorX** [comment]
On my side, upon downloading the necessary mods as well as the mod itself in a new instance, I am able to open the gui, but it appears the textures FOR the gui render invisible for unknown reasons, I will see to it be fixed in the next patch.

**Comment by decodinatorX** [comment]
I have discovered the reasoning of the invisible gui, but until I receive further information regarding this issue I will leave it open, I will however inform that the invis gui was entirely my fault and was due to the Screen Classes making use of GuiGraphics.blitInscribed as opposed to the correct one which is just GuiGraphics.blit.

**Comment by decodinatorX** [comment]
This issue has since been lengthened, as the same issue was occurring on 1.19.2 as well, however in a different fashion, in 1.19.2, I had left the Menu Types in a CLIENT env when registering proper, thus causing them to not be callable (is what I gathered).

These issues have been fixed, expect an update to 1.19.2, 1.19.3, 1.20 + 1.20.1 soon!
Thank you for your patience and report! They are greatly appreciated!
